app/config/config.yml

```diff
@@ -12,11 +12,17 @@ framework:
     form:            true
     csrf_protection: true
     validation:      { enable_annotations: true }
-    templating:      { engines: ['twig'], assets_version: 3 }
+    templating:
+        engines: ['twig']
+    assets:
+        version: 3
     default_locale:  "%locale%"
     trusted_proxies: ~
```

Comment:
The "framework.trusted_proxies" configuration key has been removed in Symfony 3.3.

Reply:
see https://symfony.com/blog/fixing-the-trusted-proxies-configuration-for-symfony-3-3